src/get-started/wordcount-example.md

```diff
@@ -479,17 +763,19 @@ This feature is not yet available in the Beam SDK for Python.
 
 ### Write Results to an Unbounded Sink
 
-When our input is unbounded, the same is true of our output `PCollection`. We need to make sure that we choose an appropriate, unbounded sink. Some output sinks support only bounded output, while others support both bounded and unbounded outputs. By using a `FilenamePolicy`, we can use `TextIO` to files that are partitioned by windows. We use a composite `PTransform` that uses such a policy internally to write a single sharded file per window.
+Since our input is unbounded, the same is true of our output `PCollection`. We need to make sure that we choose an appropriate, unbounded sink. Some output sinks support only bounded output, such as a text file. Google Cloud BigQuery is an output source that supports both bounded and unbounded input.
 
-In this example, we stream the results to a BigQuery table. The results are then formatted for a BigQuery table, and then written to BigQuery using BigQueryIO.Write.
+In this example, we stream the results to a BigQuery table. The results are then formatted for a BigQuery table, and then written to BigQuery using BigQueryIO.Write. 
 
 ```java
-  wordCounts
-      .apply(MapElements.via(new WordCount.FormatAsTextFn()))
-      .apply(new WriteOneFilePerWindow(output, options.getNumShards()));
+wordCounts.apply(ParDo.of(new FormatAsTableRowFn()))
+    .apply(BigQueryIO.Write
+      .to(getTableReference(options))
+      .withSchema(getSchema())
+      .withCreateDisposition(BigQueryIO.Write.CreateDisposition.CREATE_IF_NEEDED)
+      .withWriteDisposition(BigQueryIO.Write.WriteDisposition.WRITE_APPEND));
```

Comment:
The code should remain untouched; the paragraph above it should be corrected to indicate that it writes one file per window.